src/GraphQL/Execution/Executor.php

```diff
@@ -223,7 +223,13 @@ public function doExecute(): Promise {
     return $this->doExecuteUncached()->then(function ($result) {
       $this->context->mergeCacheMaxAge(0);
 
-      $result = new CacheableExecutionResult($result->data, $result->extensions, $result->errors);
+      // @todo keep only else case in 5.0.0.
+      if (\Drupal::config('graphql.settings')->get('deprecated_swap_errors_and_extensions_in_results')) {
```

Comment:
I think it is ok that we don't use dependency injection here to not change contructor arguments. We could use setter injection if necessary, not sure if that is worth it.

This config call is safe even if the config does not exist, tested that on my dev site where I don't have these settings yet.

Reply:
Yeah I thought about it for a moment but I felt like changing the constructor twice to accomplish this would cause more harm than it solved.

`get` indeed returns NULL for non-existent config which evaluates to `FALSE` so that opts in to the correct order (hitting the `else` branch)